code/modules/admin/topic.dm

```diff
@@ -1926,6 +1926,29 @@
 			return
 		paper_to_show.ui_interact(usr)
 
+	else if(href_list["force_cryo"])
+		if(!check_rights(R_ADMIN))
+			return
+		var/param = href_list["force_cryo"]
+		// If it's a direct reference to a mob, use that.
+		var/mob/living/target = locate(param)
+		if(!istype(target))
+			// Might be a ckey? Let's see.
+			target = get_ckey_last_living(target)
+			if(!istype(target))
+				to_chat(usr, "This can only be used on instances of type /mob/living.")
```

Comment:
all to_chat should have a span

```suggestion
				to_chat(usr, "<span class='warning'>This can only be used on instances of type /mob/living.</span>")
```